build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -32,7 +32,15 @@
         <Link>app.config</Link>
       </None>
     </ItemGroup>
+  </Target>
 
+  <!-- Add Test Explorer Service GUID required by VS, and error out if it was specified in the project file -->
+  <!-- See https://connect.microsoft.com/VisualStudio/feedback/details/800245/vs2013rc-adds-to-vs2012-c-project-section-itemgroup -->
+  <Target Name="AddTestExplorerServiceGUID">
+    <Error Condition="'%(Service.Identity)' == '{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}'" Text="Test Explorer Service GUID exists in project. It is specified in Roslyn.Toolsets.Xunit.targets" />
+    <ItemGroup>
```

Comment:
This is one of the flags that is used for various package load stuff, and I'm not sure if it works when it isn't in the proj file itself.

We should validate that TestExplorer (and anything else that depends on this flag) is properly loaded/enabled on Solution Load regardless of whether it is imported or defined directly. If importing the definition breaks things, then we shouldn't do this PR.